diff --git a/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-client.development.js b/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-client.development.js
index 0053084..b3da396 100644
--- a/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-client.development.js
+++ b/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-client.development.js
@@ -20678,8 +20678,9 @@
           (workInProgressRootRenderLanes & 62914560) ===
             workInProgressRootRenderLanes &&
           now$1() - globalMostRecentFallbackTime < FALLBACK_THROTTLE_MS)
-          ? (executionContext & RenderContext) === NoContext &&
-            prepareFreshStack(root, 0)
+          ? (executionContext & RenderContext) === NoContext
+          ? prepareFreshStack(root, 0)
+          : (workInProgressRootPingedLanes |= pingedLanes)
           : (workInProgressRootPingedLanes |= pingedLanes),
         workInProgressSuspendedRetryLanes === workInProgressRootRenderLanes &&
           (workInProgressSuspendedRetryLanes = 0));
diff --git a/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-client.production.js b/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-client.production.js
index 1f7a795..7fc570f 100644
--- a/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-client.production.js
+++ b/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-client.production.js
@@ -14034,7 +14034,9 @@ function pingSuspendedRoot(root, wakeable, pingedLanes) {
       (workInProgressRootRenderLanes & 62914560) ===
         workInProgressRootRenderLanes &&
       300 > now() - globalMostRecentFallbackTime)
-      ? 0 === (executionContext & 2) && prepareFreshStack(root, 0)
+      ? 0 === (executionContext & 2)
+          ? prepareFreshStack(root, 0)
+          : (workInProgressRootPingedLanes |= pingedLanes)
       : (workInProgressRootPingedLanes |= pingedLanes),
     workInProgressSuspendedRetryLanes === workInProgressRootRenderLanes &&
       (workInProgressSuspendedRetryLanes = 0));
diff --git a/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-profiling.development.js b/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-profiling.development.js
index ba87af2..839fcf3 100644
--- a/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-profiling.development.js
+++ b/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-profiling.development.js
@@ -20686,8 +20686,9 @@
           (workInProgressRootRenderLanes & 62914560) ===
             workInProgressRootRenderLanes &&
           now$1() - globalMostRecentFallbackTime < FALLBACK_THROTTLE_MS)
-          ? (executionContext & RenderContext) === NoContext &&
-            prepareFreshStack(root, 0)
+          ? (executionContext & RenderContext) === NoContext
+          ? prepareFreshStack(root, 0)
+          : (workInProgressRootPingedLanes |= pingedLanes)
           : (workInProgressRootPingedLanes |= pingedLanes),
         workInProgressSuspendedRetryLanes === workInProgressRootRenderLanes &&
           (workInProgressSuspendedRetryLanes = 0));
diff --git a/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-profiling.profiling.js b/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-profiling.profiling.js
index 8c82c27..d3ea41e 100644
--- a/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-profiling.profiling.js
+++ b/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-profiling.profiling.js
@@ -15728,7 +15728,9 @@ function pingSuspendedRoot(root, wakeable, pingedLanes) {
       (workInProgressRootRenderLanes & 62914560) ===
         workInProgressRootRenderLanes &&
       300 > now$1() - globalMostRecentFallbackTime)
-      ? 0 === (executionContext & 2) && prepareFreshStack(root, 0)
+      ? 0 === (executionContext & 2)
+          ? prepareFreshStack(root, 0)
+          : (workInProgressRootPingedLanes |= pingedLanes)
       : (workInProgressRootPingedLanes |= pingedLanes),
     workInProgressSuspendedRetryLanes === workInProgressRootRenderLanes &&
       (workInProgressSuspendedRetryLanes = 0));
diff --git a/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-unstable_testing.development.js b/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-unstable_testing.development.js
index 38e4149..074ba01 100644
--- a/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-unstable_testing.development.js
+++ b/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-unstable_testing.development.js
@@ -20901,8 +20901,9 @@
           (workInProgressRootRenderLanes & 62914560) ===
             workInProgressRootRenderLanes &&
           now$1() - globalMostRecentFallbackTime < FALLBACK_THROTTLE_MS)
-          ? (executionContext & RenderContext) === NoContext &&
-            prepareFreshStack(root, 0)
+          ? (executionContext & RenderContext) === NoContext
+          ? prepareFreshStack(root, 0)
+          : (workInProgressRootPingedLanes |= pingedLanes)
           : (workInProgressRootPingedLanes |= pingedLanes),
         workInProgressSuspendedRetryLanes === workInProgressRootRenderLanes &&
           (workInProgressSuspendedRetryLanes = 0));
diff --git a/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-unstable_testing.production.js b/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-unstable_testing.production.js
index 424eff7..6cd9fa6 100644
--- a/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-unstable_testing.production.js
+++ b/node_modules/next/dist/compiled/react-dom-experimental/cjs/react-dom-unstable_testing.production.js
@@ -14306,7 +14306,9 @@ function pingSuspendedRoot(root, wakeable, pingedLanes) {
       (workInProgressRootRenderLanes & 62914560) ===
         workInProgressRootRenderLanes &&
       300 > now() - globalMostRecentFallbackTime)
-      ? 0 === (executionContext & 2) && prepareFreshStack(root, 0)
+      ? 0 === (executionContext & 2)
+          ? prepareFreshStack(root, 0)
+          : (workInProgressRootPingedLanes |= pingedLanes)
       : (workInProgressRootPingedLanes |= pingedLanes),
     workInProgressSuspendedRetryLanes === workInProgressRootRenderLanes &&
       (workInProgressSuspendedRetryLanes = 0));
diff --git a/node_modules/next/dist/compiled/react-dom/cjs/react-dom-client.development.js b/node_modules/next/dist/compiled/react-dom/cjs/react-dom-client.development.js
index 4376913..06a1e15 100644
--- a/node_modules/next/dist/compiled/react-dom/cjs/react-dom-client.development.js
+++ b/node_modules/next/dist/compiled/react-dom/cjs/react-dom-client.development.js
@@ -16469,8 +16469,9 @@
           (workInProgressRootRenderLanes & 62914560) ===
             workInProgressRootRenderLanes &&
           now$1() - globalMostRecentFallbackTime < FALLBACK_THROTTLE_MS)
-          ? (executionContext & RenderContext) === NoContext &&
-            prepareFreshStack(root, 0)
+          ? (executionContext & RenderContext) === NoContext
+          ? prepareFreshStack(root, 0)
+          : (workInProgressRootPingedLanes |= pingedLanes)
           : (workInProgressRootPingedLanes |= pingedLanes),
         workInProgressSuspendedRetryLanes === workInProgressRootRenderLanes &&
           (workInProgressSuspendedRetryLanes = 0));
diff --git a/node_modules/next/dist/compiled/react-dom/cjs/react-dom-client.production.js b/node_modules/next/dist/compiled/react-dom/cjs/react-dom-client.production.js
index 755076a..ebf9847 100644
--- a/node_modules/next/dist/compiled/react-dom/cjs/react-dom-client.production.js
+++ b/node_modules/next/dist/compiled/react-dom/cjs/react-dom-client.production.js
@@ -11733,7 +11733,9 @@ function pingSuspendedRoot(root, wakeable, pingedLanes) {
       (workInProgressRootRenderLanes & 62914560) ===
         workInProgressRootRenderLanes &&
       300 > now() - globalMostRecentFallbackTime)
-      ? 0 === (executionContext & 2) && prepareFreshStack(root, 0)
+      ? 0 === (executionContext & 2)
+          ? prepareFreshStack(root, 0)
+          : (workInProgressRootPingedLanes |= pingedLanes)
       : (workInProgressRootPingedLanes |= pingedLanes),
     workInProgressSuspendedRetryLanes === workInProgressRootRenderLanes &&
       (workInProgressSuspendedRetryLanes = 0));
diff --git a/node_modules/next/dist/compiled/react-dom/cjs/react-dom-profiling.development.js b/node_modules/next/dist/compiled/react-dom/cjs/react-dom-profiling.development.js
index 43d0363..02fcfe8 100644
--- a/node_modules/next/dist/compiled/react-dom/cjs/react-dom-profiling.development.js
+++ b/node_modules/next/dist/compiled/react-dom/cjs/react-dom-profiling.development.js
@@ -16477,8 +16477,9 @@
           (workInProgressRootRenderLanes & 62914560) ===
             workInProgressRootRenderLanes &&
           now$1() - globalMostRecentFallbackTime < FALLBACK_THROTTLE_MS)
-          ? (executionContext & RenderContext) === NoContext &&
-            prepareFreshStack(root, 0)
+          ? (executionContext & RenderContext) === NoContext
+          ? prepareFreshStack(root, 0)
+          : (workInProgressRootPingedLanes |= pingedLanes)
           : (workInProgressRootPingedLanes |= pingedLanes),
         workInProgressSuspendedRetryLanes === workInProgressRootRenderLanes &&
           (workInProgressSuspendedRetryLanes = 0));
diff --git a/node_modules/next/dist/compiled/react-dom/cjs/react-dom-profiling.profiling.js b/node_modules/next/dist/compiled/react-dom/cjs/react-dom-profiling.profiling.js
index 85185b0..476b68e 100644
--- a/node_modules/next/dist/compiled/react-dom/cjs/react-dom-profiling.profiling.js
+++ b/node_modules/next/dist/compiled/react-dom/cjs/react-dom-profiling.profiling.js
@@ -12404,7 +12404,9 @@ function pingSuspendedRoot(root, wakeable, pingedLanes) {
       (workInProgressRootRenderLanes & 62914560) ===
         workInProgressRootRenderLanes &&
       300 > now$1() - globalMostRecentFallbackTime)
-      ? 0 === (executionContext & 2) && prepareFreshStack(root, 0)
+      ? 0 === (executionContext & 2)
+          ? prepareFreshStack(root, 0)
+          : (workInProgressRootPingedLanes |= pingedLanes)
       : (workInProgressRootPingedLanes |= pingedLanes),
     workInProgressSuspendedRetryLanes === workInProgressRootRenderLanes &&
       (workInProgressSuspendedRetryLanes = 0));
